lib/dialogs/import/index.tsx

```diff
@@ -8,12 +8,7 @@ import TransitionDelayEnter from '../../components/transition-delay-enter';
 import Spinner from '../../components/spinner';
 
 const SourceImporter = React.lazy(() =>
-  import(
-    /*
-    webpackChunkName: 'source-importer',
-    webpackPrefetch: true,
```

Comment:
is there an explanation for why we're removing this? I didn't see any `webpack` version changes or implications from the rest of the PR why this would need to go.

Reply:
From Slack:

```
init.ts:204 Unable to load preload script: ~/simplenote-electron/desktop/preload.js
```

I was unable to reproduce this error but I have seen similar ones before. It seems like this is probably fine to leave in but it's also the only invocation of it in the repository. The directive will preload the script during CPU idle time, meaning that imports start a hair earlier.

All that being said, while I think it's going to be better _either_ leaving this in _or_ taking it out in a separate PR with an explanation for the removal, it seems like such a small enough issue and already that it's an outlier that I don't think we need to block this PR to undo the change.